homeassistant/components/cover/zwave.py

```diff
@@ -104,43 +109,19 @@ def current_cover_position(self):
 
     def open_cover(self, **kwargs):
         """Move the roller shutter up."""
-        for value in self._node.get_values(
-                class_id=zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL).values():
-            if value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Open' or value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Up':
-                self._lozwmgr.pressButton(value.value_id)
-                break
+        self._lozwmgr.pressButton(self._open_id)
 
     def close_cover(self, **kwargs):
         """Move the roller shutter down."""
-        for value in self._node.get_values(
-                class_id=zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL).values():
-            if value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Down' or value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Close':
-                self._lozwmgr.pressButton(value.value_id)
-                break
+        self._lozwmgr.pressButton(self._close_id)
 
     def set_cover_position(self, position, **kwargs):
         """Move the roller shutter to a specific position."""
         self._node.set_dimmer(self._value.value_id, position)
 
     def stop_cover(self, **kwargs):
         """Stop the roller shutter."""
-        for value in self._node.get_values(
-                class_id=zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL).values():
-            if value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Open' or value.command_class == \
-               zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL and value.label == \
-               'Down':
-                self._lozwmgr.releaseButton(value.value_id)
-                break
+        self._lozwmgr.releaseButton(self._open_id)
 
```

Comment:
Also, before I had Open or Down as stop alternatives, because it would work on either type of devices regardless if they had Open, Close or Up, Down label pairs.